.github/CODEOWNERS

```diff
@@ -0,0 +1,70 @@
+# CODEOWNERS for ESP32 Arduino Core
+
+# This file is used to specify the code owners for the ESP32 Arduino Core.
+# Read more about CODEOWNERS:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# Note that order matters. The last matching pattern will be used.
+
+# The default owners are the active developers of the ESP32 Arduino Core.
+# Refrain from using @espressif/arduino-esp32 to avoid spamming non-developers with review requests.
```

Comment:
It's also possible to create a group or modify the list of people in arduino-esp32 group if it doesn't serve its purpose